open-source-trumps-innersource.md

```diff
@@ -0,0 +1,43 @@
+## Title
+Open Source trumps InnerSource
+
+## Patlet
+
+People find the InnerSource project but, after all things are considered, even if the InnerSource component meets their needs, they still go with the open source component.
```

Comment:
I've slightly changed the phrasing here to make it easier to understand. This reflects my understanding of the rest of the pattern. Please correct if I got the point of the pattern wrong.

```suggestion
People find an InnerSource project but, after all things are considered, even if the InnerSource component meets their needs, they still go with an equivalent open source component.
```

Reply:
That's also how I understand this pattern. Adding your suggestion to the PR.